### DISCUSSION TITLE
Para extraer datos de una página

### DISCUSSION THREAD

**Discussion by Gaeloptupus** [community/community#128530] [Programming Help]
Hola quiero saber como extraer datos de una página

**Comment by billythekid** [comment]
Hi @Gaeloptupus 👋 

I think we'll need some additional information from you to help here.

What sort of page?
What sort of data? 
Where do you want the data to go?
What tooling do you want to use?

Without this information it's not easy to answer, for example I could answer this by saying 

> To extract words[type of data] from the page of a book[what sort of page] you can use your eyes[tooling] to read it[where it goes (your brain)].

Let us know what you've already tried and a bit more about the problem you're facing and we can try to help.

**Comment by ebndev** [comment]
Hi @Gaeloptupus, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.

P.S @billythekid thanks for taking the time to answer 😄
